app/src/main/res/layout-sw320dp/activity_main_unregister.xml

```diff
@@ -66,7 +66,7 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent">
-            <fragment
+            <androidx.fragment.app.FragmentContainerView
```

Comment:
Why would it be better to use this call to a FragmentContainerView rather than directly a fragment for the map ? I know that the id refers to a container, but does that make a real difference in term of implementation ?

Reply:
From what I understand, it seems that in our case, it will not change anything. However, this is the recommended way to integrate a fragment and visibly, it allows to better manage the fragment life cycle. I also saw that we can add an android:tag which allows to find the fragment maybe more easily with FragmentManager.findFragmentByTag(String). I'm not sure if it was possible before.